lighthouse-plugin-publisher-ads/audits/first-ad-render.js

```diff
@@ -71,12 +70,7 @@ class FirstAdRender extends Audit {
     const {timing} = await ComputedAdRenderTime.request(metricData, context);
 
     if (!(timing > 0)) { // Handle NaN, etc.
-      // Currently only GPT ads are supported by this audit.
-      const nonGptAdSlots = artifacts.IFrameElements.filter(
-        (iframe) => isAdIframe(iframe) && !isGptIframe(iframe));
-      if (nonGptAdSlots.length === 0) {
-        context.LighthouseRunWarnings.push(runWarning.NoAdRendered);
-      }
+      context.LighthouseRunWarnings.push(runWarning.NoAdRendered);
```

Comment:
These changes just revert the changes to this file that were made in https://github.com/googleads/publisher-ads-lighthouse-plugin/pull/119/files, since we're now measuring based on ping time.